### ISSUE TITLE
Newest source fails build in my environment

### ISSUE THREAD

**Issue by mbkinnovo** [TheCocoaProject/cordova-plugin-nativestorage#107]
Worked fine last week... (and still does with my fork...)

```
Plugins/cordova-plugin-nativestorage/NativeStorage.m:45:8: warning: incompatible pointer to integer conversion initializing 'BOOL' (aka 'signed char') with an expression of type 'id' [-Wint-conversion]
                BOOL aBoolean = [command.arguments objectAtIndex:1];
                     ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Plugins/cordova-plugin-nativestorage/NativeStorage.m:214:32: error: expected expression
                if(reference!=nil && (NsNull*) aString == [NsNull null]) {
                                             ^
Plugins/cordova-plugin-nativestorage/NativeStorage.m:214:25: error: use of undeclared identifier 'NsNull'
                if(reference!=nil && (NsNull*) aString == [NsNull null]) {
                                      ^
Plugins/cordova-plugin-nativestorage/NativeStorage.m:223:3: error: expected expression
                else
                ^
Plugins/cordova-plugin-nativestorage/NativeStorage.m:228:3: error: expected expression
        }];
         ^
Plugins/cordova-plugin-nativestorage/NativeStorage.m:266:5: error: expected ']'
@end
    ^
Plugins/cordova-plugin-nativestorage/NativeStorage.m:266:5: error: expected '}'
Plugins/cordova-plugin-nativestorage/NativeStorage.m:208:1: note: to match this '{'
{
```

**Comment by mbkinnovo** [comment]
Seem to have to go back to 2.2.1 to have a successful build... otherwise the build fails or the ap starts with a white screen...

**Comment by alokrajiv** [comment]
cc @GillesC

**Comment by alokrajiv** [comment]
@mbkinnovo are you building from GitHub master? because we haven't updated NPM for sometime now.

**Comment by mbkinnovo** [comment]
Yeah, did build from the newest github.

**Comment by alokrajiv** [comment]
@GillesC our master is then unstable...

**Comment by alokrajiv** [comment]
Hi, is anyone else facing issue? Looks to me the master is actually stable..no other reported errors.

**Comment by ealch** [comment]
Still not working, I think that you messed up in this [commit]( https://github.com/TheCocoaProject/cordova-plugin-nativestorage/commit/8e6e6e784b006f7d51e6fe4079b59b1e67798f41).
Since  you mistyped 'NsNull' and you opened two times the clauses.

**Comment by GillesC** [comment]
Merged your PR (aff0b51cf06924d34c67425788f06c460ba3859d) thank you for your contribution! I will close this issue now :)

**Comment by alokrajiv** [comment]
Sorry about the mess up :(

thanks, @ealch & @mbkinnovo 👍 👍 👍 ... dunno how it was missed.

.... I really need to automate those tests onto Travis .

.... will squeeze in sometime in the coming weeks and get it up...so this never happens again
